cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -1,14 +1,44 @@
+import os
+
 import fauxfactory
 import pytest
 
 from cfme import test_requirements
+from cfme.infrastructure.provider import InfraProvider
 from cfme.intelligence.reports.reports import ReportDetailsView
+from cfme.markers.env_markers.provider import ONE_PER_CATEGORY
 from cfme.rest.gen_data import users as _users
+from cfme.utils.conf import cfme_data
+from cfme.utils.ftp import FTPClientWrapper
+from cfme.utils.log_validator import LogValidator
 from cfme.utils.rest import assert_response
 
 pytestmark = [test_requirements.report, pytest.mark.tier(3), pytest.mark.sauce]
 
 
+@pytest.fixture
+def get_report(appliance, request):
+    def _report(file_name, menu_name):
+        collection = appliance.collections.reports
+        file_name = "{}.yaml".format(file_name)
```

Comment:
It will be great if we wont matriculate name here `.yaml`; why not directly pass it form test? is there any spacial reason?